src/routes/safe/components/Apps/utils.ts

```diff
@@ -88,8 +94,8 @@ export const getAppInfoFromUrl = memoize(async (appUrl: string): Promise<SafeApp
 
   const appInfoData = {
     name: appInfo.name,
-    iconPath: appInfo.iconPath,
-    description: appInfo.description,
+    iconPath: appInfo?.icons?.[0]?.src || appInfo.iconPath,
```

Comment:
Is there a specific reason you choose the first element? Looking at the type, it's an array to allow for different sized icons? I'd assume you'd want to pick one of at least a specific resolution before the first one?

Reply:
The reason is we show very small icons so probably shouldn't matter which one to pick but this is a thing I'm going to discuss in the tomorrow's daily before creating a more elaborated solution (or not 😁 ... let's see). PR is in draft state right now